SOURCES/secureboot-certs

```diff
@@ -689,11 +689,6 @@ a custom auth file, or 'none' for no dbx.
 If a custom file, it must be an EFI .auth file, a DER-encoded X509 certificate,
 or a PEM X509 certificate.
 
-Choosing 'none' should be completely avoided in production systems hoping to
-benefit from Secure Boot. It renders Secure Boot practically meaningless
-because attackers may simply load any number of vulnerable binaries that were
-previously signed but later revoked, and thereby take control of the system.
-
```

Comment:
It's true that windows updates the dbx, but if you don't have a (good) dbx you may already

I still want to discourage choosing `none` unless there's a very good reason and users are aware that: they're not protected when they install an OS, 2. not all OSes bring their own dbx with them afterwards (Do Linux distributions bring it?)